cpp/ql/src/semmle/code/cpp/Linkage.qll

```diff
@@ -1,3 +1,7 @@
+/**
+ * Proivdes the `LinkTarget` class representing linker invocations at compile time.
```

Comment:
Super nit-picky suggestion: "compile time" -> "build time" or "during the build process", for consistency with the QLDoc below and because strictly speaking linking is usually done after compiling.